ports/vcmi/VCMI.sh

```diff
@@ -33,6 +33,11 @@ printf "\033c" > $CUR_TTY
 printf "\033c" > $CUR_TTY
 ## RUN SCRIPT HERE
 
+if [ -f "$GAMEDIR/libs/libicudata.so.63.bz2" ]; then
+    bzip2 -fd "$GAMEDIR/libs/libicudata.so.63.bz2"
+    rm "$GAMEDIR/libs/libicudata.so.63.bz2"
```

Comment:
This isn't necessary, `bzip2` will extract the file and delete the old one. :)

Reply:
you are right! that was unexpected for me even after checking -fd flags
fixed